src/components/forms/pattern-form/pattern-form.js

```diff
@@ -434,17 +434,19 @@ const PatternForm = ({ pattern, id, isEdit }) => {
                   {errors.additionalPrice}
                 </div>
               )}
-              <TextField
-                id='outlined-basic'
-                variant='outlined'
-                label={convertationTitle}
-                className={`
-                  ${styles.textField} 
-                  ${styles.currencyField}
+              {values.additionalPriceType === 'ABSOLUTE' ?? (
```

Comment:
?? works with null or undefined, not false
use && instead

Reply:
Fixed